frontend/src/components/toast/toast-duplicate-button.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import Center from "react-center";
+import ToastButton from "./toast-button";
+
+class ToastDuplicateButton extends React.Component {
+  constructor(props) {
+    super(props);
+    console.log("value:", this.props.value);
+    this.duplicateInput = this.duplicateInput.bind(this);
+  }
+
+  duplicateInput(event) {
+    event.preventDefault();
+    if (this.props.maxItems && this.props.maxItems <= this.props.value.length)
+      return;
+    if (this.props.onDuplicate) this.props.onDuplicate(this.props.fields);
+  }
+
+  render() {
+    const updateChildrenWithProps = (idx, d) => {
+      return React.Children.map(this.props.children, (child, i) => {
```

Comment:
you can get rid of the {} and return and just have => React.cloneElement ... as a shorthand